dds/DCPS/security/AccessControl/Governance.cpp

```diff
@@ -111,7 +111,7 @@ int Governance::load(const SSL::SignedDocument& doc)
 {
   std::string xml;
   doc.get_original_minus_smime(xml);
-  ParserPtr parser(get_parser(doc.filename(), xml));
+  ParserPtr parser(move(get_parser(doc.filename(), xml)));
```

Comment:
compiler warnings for moving a temporary... I guess clang wants you to make a named variable to move here?

Reply:
I'm kinda fed up with `unqiue_ptr` right now, I just went with pass-by-reference.